.gitignore

```diff
@@ -52,6 +52,7 @@ pytestdebug.log
 dump-temp-*
 dump-tests*
 .vim-bookmarks
+armi-venv/*
```

Comment:
Is this necessary?  It seems awfully specific to what you call your local venv.  Heck, the line below just has "venv", which you could have chosen.

Reply:
I decided to leave this because this is what name the ARMI installation instructions advices you to name the environment folder. I recommend we change the installation instructions to "venv" if we remove this change.

Reply:
@bdlafleur another strategy is to have your venv in a completely separate folder outside of your ARMI clone. This way you won't have to change the gitignore and is flexible is/when you want multiple venvs (e.g., multiple python versions, different packages installed, etc).